apps/frontend/src/lib/components/ui/tooltip/index.ts

```diff
@@ -0,0 +1,19 @@
+import Root from './tooltip.svelte';
+import Trigger from './tooltip-trigger.svelte';
+import Content from './tooltip-content.svelte';
+import Provider from './tooltip-provider.svelte';
+import Portal from './tooltip-portal.svelte';
+
+export {
+	Root,
+	Trigger,
+	Content,
+	Provider,
+	Portal,
+	//
+	Root as Tooltip,
+	Content as TooltipContent,
+	Trigger as TooltipTrigger,
+	Provider as TooltipProvider,
+	Portal as TooltipPortal
```

Comment:
Is this renaming idiomatic/common in ShadCDN? If not, might be less confusing to use original names. That'll help in silly things like grep'ing for component's _actual_ code.